product_expiry_simple/models/stock_production_lot.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class StockProductionLot(models.Model):
+    _inherit = 'stock.production.lot'
+    _rec_name = 'display_name'
+
+    expiry_date = fields.Date(string='Expiry Date')
+    display_name = fields.Char(
+        compute='compute_display_name_field',
```

Comment:
I named it that way on purpose: if you call the method compute_display_name, you have a warning in the logs of odoo v10 (this is something specific to display_name).